src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -378,6 +378,19 @@ object ICollection.SyncRoot
             }
         }
 
+        int ICollection.Count 
```

Comment:
Having spent time in codebase of this assembly, this assembly has many of these kinds of things; ie APIs that don't really make sense or not following current conventions. Unfortunately, as @AlexGhiondea said, as part of .NET Standard, these will be brought over wholesale, for better or worse.